data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1477 @@
+				accept
+
+	npc accompany save
+		government "Republic"
+		personality escort timid
+		fleet
+			names "republic" "capital"
+			variant
+				"Auxiliary (Transport)" 4
+
+	on stopover
+		event "geminus rebuilt"
+		conversation
+			`Your arrival on Millrace is similar to the arrival on Lodestone - rusted sheetmetal and spalling concrete intermingle with soot and grime to make for a fantastically unpleasant sight. Though you have to give the Syndicate credit for one thing: all their technology may be ugly, but it's efficient.`
+			`It's a familiar scene as you watch the mechanics lined up to be seated aboard their transports. Standing near one of the lines, a machinist looks over at you with a curious glance - almost like he recognizes you. It soon turns out to be the case as he asks, "Hey, aren't you that Navy Lieutenant who was on Lodestone a while back?"`
+			choice
+			`	"I have no idea what you're talking about."`
+				goto no
+			`	"I've been all over the galaxy - it's hard to say where I have and haven't been."`
+				goto maybe
+			`	"Yes, I am."`
+				goto yes
+			label no
+			`	The dockworker studies you for a long minute before he shrugs and returns his gaze forward.`
+				goto end
+			label maybe
+			`	"Pretty sure I saw you there," he comments, almost goading you on. "Maybe I'm getting you confused with the other Navy prick who was there.`
+			choice
+			`	(say nothing)`
+				goto end
+			`	(attack him)`
+			`Right as you move in to defend your honor, he procures a hidden blaster pistol from his shirt. Before you can realize your error, he suddenly shoots you several times in the stomach. You have just enough life in you to watch him be carted off by other Navy personnel before your vision is obscured by blackness.`
+			`You are dead. Not big surprise.`
+				die
+			label yes
+			`	"I thought I recognized you," he remarks with a hard look. "You wouldn't happen to know the name 'Lucas Hadley,' would you?"`
+			branch uhoh
+				has "bad look"
+			`	"Maybe. Why do you ask?"`
+				goto next
+			label uhoh
+			`Though you don't display it outwardly, you realize that must be the dockworker you almost attacked on Lodestone. Better choose your words carefully, or things might get ugly.`
+			choice
+			`	"No...why do you ask? And what does he want?"`
+				goto oops
+			`	"I know I acted out of line on Lodestone - I promise it won't happen again."`
+				goto glare
+			label oops
+			`"Justice," he says as he procures a hidden blaster pistol from his shirt. Before you can realize your error, he suddenly shoots you several times in the stomach. You have just enough life in you to watch him be carted off by other Navy personnel before your vision is obscured by blackness.`
+			`You are dead. Let this be a lesson to you - if you act like an asshole, the galaxy tends to respond in kind.`
```

Comment:
Unless you happen to be Hines.

The "Lesson" seems really heavy handed.  And considering the player is... several missions post-Hadley, loading an earlier save may not be available to "avoid the bad-end".

And why isn't there any check for "helped Hadley"?